scripts/ci_build_and_test_in_container.sh

```diff
@@ -63,7 +58,8 @@ or_die python3 scripts/config-build.py \
                -ip ${GEOSX_DIR} \
                --ninja \
                -DBLT_MPI_COMMAND_APPEND='"--allow-run-as-root;--oversubscribe"' \
-               -DGEOSX_INSTALL_SCHEMA=${GEOSX_INSTALL_SCHEMA}
+               -DGEOSX_INSTALL_SCHEMA=$([[ "$*" == *--disable-schema-deployment* ]] && echo 0 || echo 1) \
+               -DENABLE_COV=$([[ "$*" == *--test-coverage* ]] && echo 1 || echo 0)
```

Comment:
Is there a way to add code coverage when running the integrated tests?

Reply:
https://about.codecov.io/blog/how-to-set-up-codecov-with-c-and-github-actions/

Reply:
It's important to have this coverage from the integrated tests in our global figures: it's a strong part of our validation and coverage.

Reply:
@TotoGaz There is just a command that you execute under the premise of a coverage tests. For instance we are doing `make tests` right now, as the coverage tests....but we can run a script that executes both unit tests and integrateTests at a later time.